docs/usage.rst

```diff
@@ -0,0 +1,188 @@
+.. -*- coding: utf-8 -*-
+
+.. _configuration:
+
+Using Magic Folder
+==================
+
+Magic-Folder is used in two ways.  To interact with configuration, the
+``magic-folder`` command line tool is used.  For details of this, see
+the section on :ref:`config-file`.  For additional information see
+`Magic Folder CLI design`_.
+
+.. _`Magic Folder CLI design`: ../proposed/magic-folder/user-interface-design
+
+To interact with content, use your normal filesystem-based tools.  The
+folder which Magic-Folder synchronizes is a normal folder in the
+filesystem.  The platform's filesystem change notification features
+are used to detect changes.
+
+Prerequisites
+-------------
+
+You must have one or more Tahoe-LAFS client nodes configured to be
+able to store objects somewhere.  They must be able to reach their
+configured storage nodes.  The client nodes must all share the same
+storage nodes.  The nodes must be running.
+
+
+Running a Magic Folder process
+------------------------------
+
+If you are running a Tahoe-LAFS client off a Tahoe-LAFS node directory
+``alice``, like so:
+
+.. code-block:: console
+
+   $ tahoe --node-directory=alice run
+
+you can start the corresponding magic folder process like so:
+
+.. code-block:: console
+
+   $ magic-folder --node-directory=alice run --web-port tcp:6000
+
+
+Creating Magic Folders
+----------------------
+
+A new magic folder is created using the ``magic-folder create``
+command:
+
+.. code-block:: console
+
+   $ magic-folder --node-directory=alice create magic: shared-docs /home/alice/Documents/Shared
+
+A magic folder is created with an alias (such as ``magic:`` in the
+above command), a nickname (such as ``shared-docs``), and a local
+directory (such as ``~/Documents/Shared``).  The root capability for
+the new magic folder is assigned the given nickname.
+
+If the nickname and local directory are given, the client also invites
+itself to join the folder.  See `Joining a Magic Folder`_ for more
+details about this.
+
+See ``magic-folder create --help`` for specific usage details.
+
+Listing Magic Folders
+---------------------
+
+Existing magic folders can be listed using the ``magic-folder list``
+command:
+
+.. code-block:: console
+
+   $ magic-folder --node-directory=alice list
+   This client has the following magic-folders:
+     default: /home/alice/Work/Magic
+       magic: /home/alice/Documents/Shared
+
+
+Inviting Participant Devices
+----------------------------
+
+A new participant device is invited to collaborate on a magic folder
+using the ``magic-folder invite`` command:
+
+
+.. code-block:: console
+
+   $ magic-folder --node-device=alice invite magic: bob
```

Comment:
`--node-directory` instead of `--node-device`?

Reply:
Thank you!  Fixed in 29b60b4.